Some README updates (#16)
inorbit-ai/ros_amr_interop: README.md

# ROS AMR interoperability packages

> This repository is intended to host a collection of ROS packages to ease

Reviewer: ```suggestion
This repository hosts a collection of ROS packages to ease
```

the integration of ROS based robots with different interoperability
standards, with a focus on AMRs (Autonomous Mobile Robots).

> ## Packages

Reviewer: ```suggestion
## Packages in this repository
```

> 

Reviewer: I prefer to avoid having two consecutive headings without text between them. Here you could introduce the section and whatever is general about the packages.

### Mass Robotics AMR Interop Sender for ROS2

The [massrobotics_amr_sender_py](https://github.com/inorbit-ai/ros_amr_interop/tree/foxy-devel/massrobotics_amr_sender_py#readme)
package provides a ROS2 node written in Python that takes input from a
ROS2 system and publishes it to a [Mass Robotics Interop compliant
Receiver](https://github.com/MassRobotics-AMR/AMR_Interop_Standard/tree/main/MassRobotics-AMR-Receiver).

Mapping of different data elements from the ROS2 system into Mass

> Robotics Interop data format is managed through a YAML configuration

Reviewer: ```suggestion
Robotics Interop messages can be customized through a YAML configuration
```

file.

## Related Initiatives

> 

Reviewer: I'd add a sentence or two to introduce the following bullets

 * [Open-RMF](https://osrf.github.io/ros2multirobotbook/) (formerly RMF Core) is an
 open source framework based on ROS 2 to enable the interoperability of heterogeneous
 fleets of any type of robotics systems.
 * [Mass Robotics AMR Interoperability Standard](https://github.com/MassRobotics-AMR/AMR_Interop_Standard) aims to help organizations deploy AMRs from multiple vendors and have them coexist effectively.
 * [VDA 5050](https://www.vda.de/en/services/Publications/vda-5050-v-1.1.-agv-communication-interface.html)
 AGV Communications Interface describes an interface for communication between driverless
 transport vehicles (AGV) and a master control system over MQTT using standardized
 JSON messages.
 * [OPC Unified Architecture](https://opcfoundation.org/about/opc-technologies/opc-ua/)
   (OPC UA) is a machine to machine communication protocol for industrial
 automation developed by the OPC Foundation

>    * [ros_opcua_communication](http://wiki.ros.org/ros_opcua_communication) ROS bindings for different open-source OPC-UA implementations

Reviewer: I'd add a sentence after the list to recap and said that we intend to incrementally support them. The idea is that images, code blocks, lists, etc should have an intro and recap
